packages/forms/src/lib/datepicker/datepicker.module.ts

```diff
@@ -11,9 +11,9 @@ import {
 	CALENDAR_DEFAULT_MONTH_LABELS
 } from '@acpaas-ui/ngx-components/calendar';
 
-import { MaskModule } from '../mask';
+import { MaskModule } from '../mask/mask.module';
```

Comment:
Have you tried `import { MaskModule } from '@acpaas-ui/ngx-components/mask'`?

Reply:
I have now... 😉